feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/bindingadapter/TextViewBinding.kt

```diff
@@ -0,0 +1,40 @@
+package io.github.droidkaigi.confsched2019.session.ui.bindingadapter
+
+import android.graphics.Typeface
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.text.TextUtils
+import android.text.style.BackgroundColorSpan
+import android.text.style.StyleSpan
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import androidx.databinding.BindingAdapter
+import io.github.droidkaigi.confsched2019.session.R
+import java.util.regex.Pattern
+
+@BindingAdapter(value = ["app:highlightText"])
+fun setHighlightText(view: TextView, highlightText: String?) {
+    // By toString, clear highlight text.
+    val stringBuilder = SpannableStringBuilder(view.text.toString())
+    if (TextUtils.isEmpty(highlightText)) {
+        view.text = stringBuilder
+        return
+    }
+    val pattern = Pattern.compile(highlightText, Pattern.CASE_INSENSITIVE)
+    val matcher = pattern.matcher(view.text)
+    while (matcher.find()) {
+        stringBuilder.setSpan(
+            BackgroundColorSpan(ContextCompat.getColor(view.context, R.color.colorSecondary)),
```

Comment:
Can you create this and use it?
`<color name="colorSecondary_50">#80E6650F</color>`

Reply:
you got it 👍